Docs/Architecture Decision Records/001 - PlainTextDeserializationVisitor.md

```diff
@@ -0,0 +1,22 @@
+# 1. Support "deserialization" of plain text 
```

Comment:
I would call the folder as `/Docs/architecture/decisions` and also with this we unify how we defined in framework bundle